main.nf

```diff
@@ -53,15 +53,15 @@
  */
 
 // Pipeline version
-version = 0.2
+version = 0.1
 
 // Reference genome index
 params.genome = 'GRCh37'
 params.index = params.genomes[ params.genome ].star
 params.gtf   = params.genomes[ params.genome ].gtf
 params.bed12 = params.genomes[ params.genome ].bed12
 
-single='testing'
+single='null'
```

Comment:
`null` not `'null'` (null type rather than a string). Not that it matters. But, you know, I'm a bit OCD.